test/unit/css.js

```diff
@@ -1323,11 +1323,11 @@ QUnit.test( "Keep the last style if the new one isn't recognized by the browser
 	assert.equal( el.css( "position" ), "absolute", "The old style is kept when setting an unrecognized value" );
 	el = jQuery( "<div></div>" ).css( "position", "absolute" ).css( "position", " " );
 
-	// Support: Edge 14
+	// Support: Edge 14-15
 	// Edge collapses whitespace-only values when setting a style property and
 	// there is no easy way for us to work around it. Just skip the test there
 	// and hope for the better future.
-	if ( /edge\//i.test( navigator.userAgent ) ) {
+	if ( /\bedge\//i.test( navigator.userAgent ) ) {
 		assert.ok( true, "Skipped (Edge 14 handles whitespace-only values incorrectly)" );
```

Comment:
Please pull this out into its own `QUnit[ /\bedge\//i.test( navigator.userAgent ) ? "skip" : "test" ]`.